README.md

```diff
@@ -4,44 +4,62 @@
 [![star this repo](http://githubbadges.com/star.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java)
 [![fork this repo](http://githubbadges.com/fork.svg?user=vesoft-inc&repo=nebula-java&style=default)](https://github.com/vesoft-inc/nebula-java/fork)
 
-This guide provides instructions and options for connecting **Nebula Graph2.0** for Java developer. However, Nebula Java is not thread-safe.
+Nebula Java is a Java client for Java developers to connect their projects to Nebula Graph.
+
+> **NOTE**: Nebula Java is not thread-safe.
```

Comment:
I'm not sure if 2.0-client is not thread-safe.

Reply:
> I'm not sure if 2.0-client is not thread-safe.

yes, now is also not thread-safe.